File: java-main/java-main/Switch/p5.java
class Demo{

        public static void main(String[]args){

                int x=4;
		int a=1;
		int b=2;

                switch(x){


                        case a:
                                System.out.println("One");
				break;
                        case b:
                                System.out.println("Two");
				break;
                        case a+b:
                                System.out.println("Three");
				break;
                        case a+a+b:
                                System.out.println("Four");
				break;
			//constant expression required
			case a+a+a+b:
				     System.out.println("Five");
				break;
                        default :
                                System.out.println("No Match");
				break;
                }
                System.out.println("After switch");
        }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/practical1/p5.c
/* wap that takes a number from 0 to 5 and print its spelling*/
#include<stdio.h>
void main(){

        int x;

        printf("Enter number\n");
        scanf("%d",&x);

        switch(x){
		case 0:
			printf("Zero\n");
			break;

                case 1:
                        printf("One\n");
                        break;
                case 2:
                        printf("Two\n");
                        break;
                case 3:
                        printf("Three\n");
                        break;
                case 4:
                        printf("Four\n");
                        break;
                case 5:
                        printf("Five\n");
			break;
		default:
			printf("Wrong input\n");
	}
}
